openmc/cell.py

```diff
@@ -219,6 +225,14 @@ def paths(self):
         return self._paths
 
     @property
+    def bounding_box(self):
```

Comment:
It is indeed. It's mostly a convenience thing, but it is used in `openmc.Plot.from_geometry` which uses `Geometry.bounding_box`  which uses `Universe.bounding_box` which uses `Cell.bounding_box`.